virtme/guest/virtme-init

```diff
@@ -391,11 +393,13 @@ if [[ ! -e "/dev/$consdev" ]]; then
 fi
 
 # Redirect current stdout/stderr to consdev
-exec 1>/dev/${consdev}
+exec 1>/dev/"${consdev}"
```

Comment:
(detail, maybe with the double quotes around the whole string: `"/dev/${consdev}"`?)

Reply:
Can change it, but doesn't make any difference.